fixes.inc

```diff
@@ -6836,6 +6917,16 @@ native BAD_SetPlayerName(playerid, const name[]) = SetPlayerName;
 		//  END:   TogglePlayerControllable
 		// =================================
 
+	  	// =========================
+		// BEGIND: SilentTeleport
+		// =========================
+		#if FIX_SilentTeleport
```

Comment:
You forgot to add check for definition of `FIX_SilentTeleport` before `OnPlayerSpawn`.

Reply:
Done.